core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/DocFormatter.java

```diff
@@ -53,8 +53,8 @@ default String getAnchor(String string) {
 
         // Apostrophes.
         string = string.replaceAll("([a-z])'s([^a-z])", "$1s$2");
-        // Allow only letters, -, _, .
-        string = string.replaceAll("[^\\w-_\\.]", "-").replaceAll("-{2,}", "-");
+        // Allow only letters, -, _
+        string = string.replaceAll("[^\\w-_]", "-").replaceAll("-{2,}", "-");
```

Comment:
So we're breaking every single link pointing to Quarkus configuration properties from other websites? I left quite a few of those out there, on stackoverflow, Zulip, Discourse, GitHub, ...

Can we add a secondary anchor for the deprecated, dot-containing id?